packages/ember-glimmer/tests/integration/components/curly-components-test.js

```diff
@@ -413,6 +413,26 @@ moduleFor('Components test: curly components', class extends RenderingTest {
     assert.equal($span.attr('class'), 'inner');
   }
 
+  ['@test a void element does not have childViews'](assert) {
+    let fooBarInstance;
+    let FooBarComponent = Component.extend({
+      init() {
+        this._super();
+        fooBarInstance = this;
+      }
+    });
+
+    this.registerComponent('foo-bar', { ComponentClass: FooBarComponent, template: '{{yield}}' });
+
+    this.render('{{foo-bar}}');
+
+    assert.deepEqual(fooBarInstance.childViews.length, 0);
```

Comment:
The original test was checking for `childNodes` of the components element, not for `childViews.

I think this (and the one below) should be:

```js
assert.deepEqual(fooBarInstance.element.childNodes.length, 0, 'no childNodes are added for `<input>`');
```